flutter_application/lib/ServicePage/action_reaction_page.dart

```diff
@@ -14,8 +14,8 @@ String chooseImageServiceAction(actionServiceChoose) {
   if (actionServiceChoose == "Youtube") {
     res = "assets/images/Youtube-Symbole.jpg";
   }
-  if (actionServiceChoose == "Planner") {
-    res = "assets/images/Planner-logo.jpg";
+  if (actionServiceChoose == "Facebook") {
+    res = "assets/images/Facebook-logo.jpg";
```

Comment:
delete planner logo which is useless now

Reply:
no because we change planner to facebook and here it's for the logo of the action/reaction link